build.gradle

```diff
@@ -203,14 +216,20 @@ task generateInstallers(type: Install4jTask, dependsOn: [renameShadowJar, copyJR
     }
 }
 
-task setRunnableLinux(dependsOn: [generateInstallers]) {
+task prepareInstallers(group: 'release', dependsOn: [generateInstallers]) {
     doLast {
-        def linuxFile = "TripleA_${version}_unix.sh"
-        ant.chmod(dir: "$buildDir/releases", perm: '+x', includes: linuxFile)
+        def releasesDir = "$buildDir/releases"
+
+        ant.chmod(dir: releasesDir, perm: '+x', includes: "TripleA_${version}_unix.sh")
```

Comment:
Yes, install4j (at least on Linux) does not automatically set the executable bit for the Unix installer.

Actually, I interpreted your suggestion as checking to see if install4j provided an option to set the Unix mode of the generated media file.  Unfortunately, while install4j does provide the ability to set the Unix mode for the installed files themselves, it does not appear to support this feature for the actual installer.

However, I found an option that allows us to override the media file name.  So I replaced the Gradle code that renamed the Windows installers with custom media file names in the install4j build script (see the `mediaFileName` attribute on the Windows 32/64 children of the `mediaSets` element).  So, in the end, it was worth the investigation. :smile:  bd000e6